crates/precompile/src/secp256k1.rs

```diff
@@ -40,7 +40,7 @@ mod secp256k1 {
 
 #[cfg(feature = "secp256k1")]
 #[allow(clippy::module_inception)]
-mod secp256k1 {
+pub mod secp256k1 {
```

Comment:
why do you need to be public? This mod is mad to hide the secp256k1 and k256 inclusion.

`ecrecover` should be made publish

Reply:
makes total sense, with these changes https://github.com/bluealloy/revm/pull/1201/commits/746039da228d75a2fe9eb7bcb1e35046b4227477  `ecrecover` is reexported, PTAL